i18n/zh-CN/docusaurus-plugin-content-blog/2024-03-04-kcl-0.8.0-release/index.md

```diff
@@ -0,0 +1,354 @@
+
+[KCL](https://github.com/kcl-lang/kcl) 是一个 CNCF 基金会托管的面向云原生领域开源的基于约束的记录及函数编程语言，期望通过成熟的编程语言技术和实践来改进对大量繁杂配置比如云原生 Kubernetes 配置场景的编写，致力于围绕配置的模块化、扩展性和稳定性，打造更简单的逻辑编写体验，构建更简单的自动化和生态集成路径。
+
+## 语言更新
+
+### 😸 新增 Linux arm64 版本
+
+KCL 的 Release 产物中新增了对 Linux arm64 平台的支持。
+
+可以在 [KCL Release Page](https://github.com/kcl-lang/kcl/releases) 中找到后缀为 `linux-arm64` 的压缩包。
+
+### 🔧 诊断信息的优化
+
+KCL 语法在 if 块中使用的是 `elif` 关键字，而不是 `else if`。
+
+编译以下 KCL 程序：
+
+```kcl
+if True: a = 1
+else if False: b = 1
+```
+
+KCL 在诊断信息中增加了错误修正的建议：
+
+```shell
+error[E1001]: InvalidSyntax
+ --> main.k:2:6
+  |
+2 | else if False: b = 1
+  |      ^ 'else if' here is invalid in KCL, consider using the 'elif' keyword
+  |
+```
+
+### 🚀 语言编写体验优化
+
+#### KCL 标准库新增文件系统访问方法
+
+KCL 增加了访问文件系统的方法。在 v0.8.0 版本中支持了包括 `read`, `glob` 等访问文件系统的方法。
+
+通过 `read` 方法，可以读取一个文件内容为字符串。
+
+```kcl
+import file
+
+a = read("hello.txt")
+```
+
+在文件 `hello.txt` 中添加如下内容：
+
+```shell
+Hello World !
+```
+
+编译结果
+
+```shell
+a: Hello World !
+```
+
+通过结合 `json.decode` 方法，可以轻松的实现将 json 文件反序列化。
+
+在 `hello.json` 文件中增加如下内容
+
+```json
+{
+  "name": "John",
+  "age": 10
+}
+```
+
+KCL 程序如下所示：
+
+```kcl
+import file
+import json
+
+_a = json.decode(file.read("hello.json"))
+
+name = _a.name
+age = _a.age
+```
+
+编译结果如下所示：
+
+```shell
+name: John
+age: 10
+```
+
+更多内容 - [https://kcl-lang.io/zh-CN/docs/reference/model/file/](https://kcl-lang.io/zh-CN/docs/reference/model/file/)
+
+#### KCL 编译缓存路径支持使用环境变量 KCL_CACHE_PATH 指定
+
+KCL 编译器会将缓存生成到环境变量`KCL_CACHE_PATH`指定的目录当中，如果没指定，将会生成到项目根目录中。
+
+#### 插件系统支持使用 golang 编写 KCL 插件
+
+使用 golang 定义 hello 插件。
+
+```golang
+package hello_plugin
+
+import (
+    "kcl-lang.io/kcl-go/pkg/plugin"
+)
+
+func init() {
+    plugin.RegisterPlugin(plugin.Plugin{
+        Name: "hello",
+        MethodMap: map[string]plugin.MethodSpec{
+            "add": {
+                Body: func(args *plugin.MethodArgs) (*plugin.MethodResult, error) {
+                    v := args.IntArg(0) + args.IntArg(1)
+                    return &plugin.MethodResult{V: v}, nil
+                },
+            },
+        },
+    })
+}
+```
+
+借助 kcl-go 开发，扩展 KCL 编译器使用插件。
+
+```kcl
+package main
+
+import (
+    "fmt"
+
+    "kcl-lang.io/kcl-go/pkg/kcl"
+    "kcl-lang.io/kcl-go/pkg/native"                // Import the native API
+    _ "kcl-lang.io/kcl-go/pkg/plugin/hello_plugin" // Import the hello plugin
+)
+
+func main() {
+    // Note we use `native.MustRun` here instead of `kcl.MustRun`, because it needs the cgo feature.
+    yaml := native.MustRun("main.k", kcl.WithCode(code)).GetRawYamlResult()
+    fmt.Println(yaml)
+}
+
+const code = `
+import kcl_plugin.hello
+
+name = "kcl"
+three = hello.add(1,2) # 3
+```
+
+### 🏄 SDK & API 更新
+
+
+TODO
+
+### 🐞 其他更新及错误修复
+
+TODO
+
+## IDE & 工具链更新
+
+### IDE 更新
+
+体验改进
+支持增量解析和异步编译功能，提升性能
+错误修复
+修复 assert 语句中字符串插值变量不能跳转的异常
+修复了字符串中异常触发函数补全的异常
+修复 import 语句别名语义检查和补全的异常
+修复了 schema 中 check 表达式补全的异常
+
+TODO
```

Comment:
We can use some GIFs to showcase new IDE features, cc @zong-zhe @He1pa

Reply:
fixed